nuttx/arch/arm/Kconfig

```diff
@@ -326,25 +125,16 @@ config ARMV7M_CMNVECTOR
 		architectures.
 
 config ARCH_HAVE_FPU
-	bool
-	default n
+        bool 
+	default y
 
 config ARCH_FPU
-	bool "FPU support"
+        bool
 	default y
-	depends on ARCH_HAVE_FPU
-	---help---
-		Build in support for the ARM Cortex-M4 Floating Point Unit (FPU).
-		Check your chip specifications first; not all Cortex-M4 chips support the FPU.
 
 config ARMV7M_MPU
-	bool "MPU support"
```

Comment:
I don't get this at all.  You don't want to select that it has an MPU or tell people about it?